HarpGuru/HarpFace/components/index.tsx

```diff
@@ -2,21 +2,20 @@ import { useGlobal } from 'reactn'
 import { View } from 'react-native'
 import React from 'react'
 
-import type { HarpFaceProps } from '../types'
 import { getStyles } from '../styles'
 import { getHarpFaceFacts } from '../helpers'
 import { HarpFaceFragment } from '../HarpFaceFragment'
 
-export const HarpFace = (props: HarpFaceProps): React.ReactElement => {
+export const HarpFace = (): React.ReactElement => {
   const [activeHarpStrata] = useGlobal('activeHarpStrata')
-  const styles = getStyles({ ...props, activeHarpStrata })
+  const [activeDisplayMode] = useGlobal('activeDisplayMode')
+  const styles = getStyles({ activeHarpStrata })
```

Comment:
It's not really necessary to be passing this param as an object now is it. A basic param would do since there's just the one of them.

Reply:
Referenced in open issue: https://github.com/js-jslog/harpnative/issues/45#issuecomment-663618566